pkgs/tools/text/sorted-grep/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "sorted-grep";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/sgrep/sgrep-${version}.tgz";
+    hash = "sha256-3F7cXrZnB38YwE1sHYm/CIGKmG+1c0QU+Pk3Y53a0T4=";
+  };
+
+  postPatch = ''
+    # Its Makefile is missing compiler flags and an install step
+    rm -f Makefile
+  '';
+
+  buildPhase = ''
+    $CC -Wall -O2 -o sgrep sgrep.c
+  '';
+
+  installPhase = ''
```

Comment:
Don't forget the decorations in the customized phases.

Reply:
Thanks, added